### ISSUE TITLE
Info missing in encyclopedia

### ISSUE THREAD

**Issue by jusah123** [R-Vaccari/bannerlord-banner-kings#63]
Have tried this with every mod combo possible, when Banner Kings is loaded with another mod that changes the encyclopedia, it removes segments from it including age and culture. No other mods, including those that change the encyclopedia, do this.

**Comment by enisn** [comment]
yeah, the info section is missing and I spent a half hour to find which mod cause it
![image](https://user-images.githubusercontent.com/23705418/228043301-d4fb3817-eb92-41c4-a928-83f63a50eebd.png)

**Comment by laryakan** [comment]
Issue persists on latest BK version (1.2.7), Bannerlord 1.1.2
![image](https://user-images.githubusercontent.com/6781104/228652763-3876e82e-7c41-48be-97ed-3b71f2a42aa9.png)

**Comment by EdmundNasher** [comment]
Issue fixed.

**Comment by enisn** [comment]
Fixed by https://github.com/R-Vaccari/bannerlord-banner-kings/commit/3dbf5f8ca3ab4d3e4c8ae2a426f399a79af1c75d
